editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-contexts.tsx

```diff
@@ -49,11 +50,14 @@ interface UtopiaProjectContextProps {
   projectContents: ProjectContentTreeRoot
   openStoryboardFilePathKILLME: string | null
   transientFileState: TransientFileState | null
+  resolve: (importOrigin: string, toImport: string) => Either<string, string>
 }
+const EmptyResolve = (): Either<string, string> => left('')
```

Comment:
Should there be an error message in that string?

Reply:
This definitely feels wrong to me too.

Reply:
fixed